make/photon/notary/server-start.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-chown 10000:10000 -R /config
-sudo -E -u \#10000 sh -c "/usr/bin/env /migrations/migrate.sh && /bin/notary-server -config=/config/server-config.json -logf=logfmt"
+chown 10000:10000 -R /etc/notary/private
```

Comment:
Will we save some thing to /etc/notary/private ?   Or this is the default directory for notary?